listener/upload.go

```diff
@@ -178,6 +181,7 @@ func updateSystemPlatform(tx *gorm.DB, inventoryID string, accountID int, host *
 		StaleTimestamp:        optParseTimestamp(host.StaleTimestamp),
 		StaleWarningTimestamp: optParseTimestamp(host.StaleWarningTimestamp),
 		CulledTimestamp:       optParseTimestamp(host.CulledTimestamp),
+		Stale:                 staleWarning != nil && staleWarning.Before(time.Now()),
```

Comment:
Is it good place to update Stale timestamp? I thought stale systems are those we don't get uploads for.

Reply:
Probably, only other place where we save systems is at the end of evaluation.

Reply:
Stale systems are the ones, which haven't received upload in a while. That does not mean they can't get uploaded.